src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java

```diff
@@ -334,40 +332,37 @@ public boolean isWithinGameField(Point p){
      */
     public List<Pair<Coordinate, Double>> getWorstPointsOnAllEdges(double minDistance) {
 
-
         // making the decision transparent by drawing all lines which are inspected
         Geometry innerBufferedArea = possibleArea.getObject().buffer(-0.001);
 
         // due to rounding errors in the calculation a minimally negative buffer is to be drawn
         //innerBufferItem = new GeometryItem(innerBufferedArea.copy(), GeometryType.INNER_BUFFER, innerBufferStyle);
```

Comment:
inline comments (@axel1200 screeching)

Reply:
sorry, I dont wrote this ^^ but I could add a PR, deleting inline comments